platform/android/java/app/AndroidManifest.xml

```diff
@@ -20,14 +17,11 @@
         android:glEsVersion="0x00020000"
         android:required="true" />
 
-<!-- Custom user permissions XML added by add-ons. It's recommended to add them from the export preset, though. -->
-<!--CHUNK_USER_PERMISSIONS_BEGIN-->
-<!--CHUNK_USER_PERMISSIONS_END-->
-
-    <!-- Any tag in this line after android:icon will be erased when doing custom builds. -->
-    <!-- If you want to add tags manually, do before it. -->
-    <!-- WARNING: This should stay on a single line until the parsing code is improved. See GH-32414. -->
-    <application android:label="@string/godot_project_name_string" android:allowBackup="false" tools:ignore="GoogleAppIndexingWarning" android:icon="@mipmap/icon" >
+    <application
+		android:label="@string/godot_project_name_string"
```

Comment:
Mix of tabs and spaces in this file. The original one used 4 spaces for indentation.

Reply:
Android Studio was picking the `.editorconfig` file and updated the file indents based on the default configuration (tab). 
I've added another entry in that file to use spaces as indent for `AndroidManifest.xml` and `*.gradle` files.